src/ccutil/tessdatamanager.cpp

```diff
@@ -245,11 +245,11 @@ bool TessdataManager::CombineDataFiles(
     const char *language_data_path_prefix,
     const char *output_filename) {
   // Load individual tessdata components from files.
-  for (unsigned i = 0; i < TESSDATA_NUM_ENTRIES; ++i) {
+  for (auto kTessdataFileSuffixe : kTessdataFileSuffixes) {
```

Comment:
**kTessdataFileSuffixe**

kSomthing is way to write consts.

Should be tessdata_file_suffix or something shorter.

Reply:
Fix now, thank you.